pwnlib/elf/elf.py

```diff
@@ -1328,13 +1328,23 @@ def relro(self):
         .. _DF_BIND_NOW: http://refspecs.linuxbase.org/elf/gabi4+/ch5.dynamic.html#df_bind_now
 
         """
+        if not any('GNU_RELRO' in str(s.header.p_type) for s in self.segments):
```

Comment:
Generating the binaries on the fly is not easy, as most distros have implicit linker flags. I couldn't generate a non-relro binary on neither Gentoo nor Ubuntu 16.04, so I ended up using Debian 8.